manifest.yaml

```diff
@@ -62,19 +62,19 @@ health-checks:
     args: []
     # When `inject` is true, the health check will use the main image to run the health check. This is faster as there is no need to spin up an additional docker container
     # When `inject` is false, the health check will use whatever image is specified. This is useful when using a system image with additional utilities to run a health check. If inject=false, then system must equal true
-    inject: true
+#    inject: true
     # Optional if false - indicates if an image that is preloaded onto the system will be used
     system: false
     # Required - valid values are yaml, toml, json
     io-format: json
+config: ~
+properties: ~
+#  type: script
 # This denotes any data, asset, or pointer volumes that should be connected when the "docker run" command is invoked
 volumes:
   # This is the image where files from the project asset directory will go
   main:
     type: data
-  # This is an example of an asset volume
-  compat:
-    type: assets 
```

Comment:
Either decide to leave in examples even if they aren't used (like this, and migrations) - or remove them completely. 

I'm fine with either way, but think having them included would be helpful to reference format.